sgkit/stats/aggregation.py

```diff
@@ -0,0 +1,55 @@
+import xarray as xr
+from xarray import DataArray, Dataset
+
+
+def allele_count(ds: Dataset) -> DataArray:
+    """Compute allele count from genotype calls
+
+    Parameters
+    ----------
+    ds : Dataset
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`
+
+    Examples
+    --------
+
+    >>> import sgkit as sg
+    >>> from sgkit.testing import simulate_genotype_call_dataset
+    >>> ds = simulate_genotype_call_dataset(n_variant=3, n_sample=2, seed=1)
+    # TODO: Should we expose visual convenience functions like this?
+    >>> ds['call/genotype'].to_series().unstack().astype(str).apply('/'.join, axis=1).unstack()
+    samples 0   1
+    variants
+    0       1/0	1/0
+    1       1/0	1/1
+    2       0/1	1/0
+    3       0/0	0/0
+
+    >>> allele_count(ds)
+    <xarray.DataArray 'variant/allele_count' (variants: 4, alleles: 2)>
+    array([[2, 2],
+        [1, 3],
+        [2, 2],
+        [4, 0]])
+    Dimensions without coordinates: variants, alleles
+
+    Returns
+    -------
+    DataArray
```

Comment:
```suggestion
    variant/allele_count : DataArray
```

Could put the name here?

Reply:
Changed in https://github.com/pystatgen/sgkit/pull/36/commits/885a50d30f72d0c81ce56e01e3a13975fac9d02d